### PR TITLE
BXMSDOC-4560 introduce datasource-management.properties and security-management.properties in the document Installing and configuring Red Hat Process Automation Manager on Red Hat JBoss EAP

### PR DESCRIPTION
**Pull request by emmurphy1**
Details in jira:
[BXMSDOC-4560](https://issues.jboss.org/browse/BXMSDOC-4560)

### CODE REVIEW

File: doc-content/enterprise-only/installation/eap-ba-dm-run-proc.adoc [hunk 1]
```diff
@@ -5,6 +5,9 @@ Use this procedure to run the {PRODUCT} on {EAP} in standalone mode.
 .Prerequisites
 * {PRODUCT} is installed and configured.
 
+NOTE: If you change the default host (`localhost`) or the default port (`9990`) , you must edit the `business-central.war/WEB-INF/classes/datasource-management.properties`
+or `business-central.war/WEB-INF/classes/security-management.properties` file as described in https://access.redhat.com/solutions/3519551[Solution 3519551] before you run {PRODUCT}.
```
Comment (sutaakar) [inline doc-content/enterprise-only/installation/eap-ba-dm-run-proc.adoc]:
```suggestion
and `business-central.war/WEB-INF/classes/security-management.properties` file as described in https://access.redhat.com/solutions/3519551[Solution 3519551] before you run {PRODUCT}.
```

**Comment by sutaakar** [comment]
one minor correction, otherwise looks good.

File: doc-content/enterprise-only/installation/eap-ba-dm-run-proc.adoc [hunk 2]
```diff
@@ -5,6 +5,9 @@ Use this procedure to run the {PRODUCT} on {EAP} in standalone mode.
 .Prerequisites
 * {PRODUCT} is installed and configured.
 
+NOTE: If you changed the default host (`localhost`) or the default port (`9990`), you must edit the `business-central.war/WEB-INF/classes/datasource-management.properties`
+and `business-central.war/WEB-INF/classes/security-management.properties` files as described in https://access.redhat.com/solutions/3519551[Solution 3519551] before you run {PRODUCT}.
+
```
Comment (sterobin) [inline doc-content/enterprise-only/installation/eap-ba-dm-run-proc.adoc:10]: Suggest "If you changed ..., then before you run {PRODUCT}, you must edit the ..." It's clearest imo to begin with when something is done, then what is done. "After you X, ..."  "Before you X, ..."  Some people argue that the syntax should just mirror the order in which you actually do the thing, so "After you X, ..." but "Do W before X." But this doesn't work in long constructs like this one, where the "before" bit is way down the road, plus even in short cases, I still believe it's clearest to state the temporal constraint and then the action. TMI, I know, but just so you don't think it's entirely just subjective.

**Review by sterobin — COMMENTED** [review]
@emmurphy1 Done. Trivial comment.

**Pull request merged**